src/config.rs

```diff
@@ -200,30 +231,48 @@ impl ConfigBuilder {
     }
 
     /// Set time format string to use rfc2822.
+    #[cfg(not(feature = "chrono"))]
     pub fn set_time_format_rfc2822(&mut self) -> &mut ConfigBuilder {
         self.0.time_format = TimeFormat::Rfc2822;
         self
     }
 
     /// Set time format string to use rfc3339.
+    #[cfg(not(feature = "chrono"))]
     pub fn set_time_format_rfc3339(&mut self) -> &mut ConfigBuilder {
         self.0.time_format = TimeFormat::Rfc3339;
         self
     }
 
+    /// Set offset used for logging time (default is 0)
+    #[cfg(feature = "chrono")]
+    pub fn set_time_offset(&mut self, time_offset: FixedOffset) -> &mut ConfigBuilder {
+        self.0.time_offset = time_offset;
+        self
+    }
+
     /// Set offset used for logging time (default is UTC)
+    #[cfg(not(feature = "chrono"))]
     pub fn set_time_offset(&mut self, offset: UtcOffset) -> &mut ConfigBuilder {
         self.0.time_offset = offset;
         self
     }
 
+    /// set if you log in local timezone or UTC (default is UTC)
```

Comment:
This is possibly a little confusing, because users setting an offset would not expect UTC or the Local timezone and it sounds like these two options are exclusive.

Rather this should state, that it changes the *relative* timezone before applying the offset.